fava/templates/source.html

```diff
@@ -15,7 +15,7 @@
                 </tr>
             </thead>
             <tbody>
-                {% for source_file in api.source_files() %}
+                {% for source_file in api.source_files()|sort %}
```

Comment:
The API should be consistent in what it returns, nonetheless, so I think it's worth changing.